Analyzer panic handling a query

Using ES for the backend, asking for a previos graph with a query like
```
G.At(1592825181).V().Has('Type', 'Category').HasEither('Description', 'Operaciones',).Out
```

Analyzer crashes with:
```
panic: assignment to entry in nil map

goroutine 10753551 [running]:
github.com/skydive-project/skydive/graffiti/graph/traversal.(*GraphTraversalV).As(...)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/graph/traversal/traversal.go:755
github.com/skydive-project/skydive/graffiti/graph/traversal.(*GremlinTraversalStepAs).Exec(0xc0008d0560, 0x5654040, 0xc00075a990, 0x56542c0, 0xc0008d0580, 0x0, 0x0)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/graph/traversal/traversal_parser.go:824 +0x13f
github.com/skydive-project/skydive/graffiti/graph/traversal.(*GremlinTraversalSequence).Exec(0xc001400700, 0xc0003bff20, 0x0, 0x0, 0x0, 0xffffffffffffffff, 0xffffffffffffffff)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/graph/traversal/traversal_parser.go:892 +0x179
github.com/skydive-project/skydive/graffiti/common.(*SubscriberEndpoint).getGraph(0xc0004acaa0, 0xc000153600, 0x83f, 0xc001400700, 0x0, 0x0, 0x8, 0x13)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/common/subscriber_endpoint.go:53 +0x46
github.com/skydive-project/skydive/graffiti/common.(*SubscriberEndpoint).newSubscriber(0xc0004acaa0, 0xc002e10ae0, 0x12, 0xc000153600, 0x83f, 0x0, 0x0, 0x10, 0x10)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/common/subscriber_endpoint.go:72 +0xdd
github.com/skydive-project/skydive/graffiti/common.(*SubscriberEndpoint).OnStructMessage(0xc0004acaa0, 0x56c2c40, 0xc001157f40, 0xc000a637c0)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/common/subscriber_endpoint.go:140 +0x676
github.com/skydive-project/skydive/graffiti/websocket.(*structSpeakerEventDispatcher).dispatchMessage(0xc000c49740, 0xc001157f40, 0xc000a637c0)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/websocket/message.go:273 +0x247
github.com/skydive-project/skydive/graffiti/websocket.(*StructSpeaker).OnMessage(0xc001157f40, 0x56c2c40, 0xc001157f40, 0x55ab7a0, 0xc0030bff20)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/websocket/message.go:389 +0x1db
github.com/skydive-project/skydive/graffiti/websocket.(*Conn).run.func2(0xc000187000, 0x884, 0xe00, 0x0, 0x0)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/websocket/client.go:355 +0xe2
github.com/skydive-project/skydive/graffiti/websocket.(*Conn).run.func5(0xc00067de40, 0xc00175ea20, 0xc00208f710)
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/websocket/client.go:399 +0x9c
created by github.com/skydive-project/skydive/graffiti/websocket.(*Conn).run
        /home/adrian/go/src/github.com/skydive-project/skydive/graffiti/websocket/client.go:395 +0x1a3
```

--- a/graffiti/graph/traversal/traversal.go
+++ b/graffiti/graph/traversal/traversal.go
@@ -562,7 +562,10 @@ func (t *GraphTraversal) Context(s ...interface{}) *GraphTraversal {
 		return &GraphTraversal{error: err}
 	}
 
-	return &GraphTraversal{Graph: g}
+	return &GraphTraversal{
+		Graph: g,
+		as:    make(map[string]*GraphTraversalAs),
+	}
 }
 
 // V step : [node ID]

